bucket/steamcmd.json

```
@@ -13,13 +13,13 @@
     ],
     "checkver": {
         "url": "http://media2.steampowered.com/client/steam_cmd_win32",
-        "regex": "\"version\"\\s+\"(?<version>\\d+)\"[\\s\\S]*steamcmd_win32\\.zip\\.(?<random>155fa64ee20d61958f49ac0b813d9a6d00f8f82d)"
+        "regex": "\"version\"\\s+\"(?<version>\\d+)\"[\\s\\S]*steamcmd_win32\\.zip\\.(?<random>[a-z\\d]{40})"
```

```suggestion
        "regex": "\"version\"\\s+\"(\\d+)\"[\\s\\S]*?steamcmd_win32\\.zip\\.(?<random>[a-f\\d]{40})"
```

The ${random} is indeed file's SHA-1 hash.